### PR TITLE
Add extension description toggle button

### LINKED ISSUE #164: Add extension description and toggle button to extensions list
Having details on each extension might be easier for filter and/or user choosing the extension he needs.
It will certainly increase the clutter in the UI so there should be a preference to keep the current spartan look.
Bonus point if we can have a button on the top right to toggle details dynamically

### PR DESCRIPTION
**Pull request by xorye**
This PR is not ready yet.
When it is ready, the PR should fix #164. 

This PR adds a button on the top right that toggles the extensions selection list. The button's colours are different for a dark and light theme.
demo gif

Currently, clicking the button while you are scrolled all the way down, would bring you back to the top. The VS Code api does not provide a clear way to fix this.

This PR also introduces a user config setting to set whether or not the descriptions should appear or not.

Signed-off-by: David Kwon <dakwon@redhat.com>

### CODE REVIEW

**Comment by xorye** [comment]
This PR is ready for review. 

As an aside: the icon's SVG was from this website: https://feathericons.com/?query=info

File: README.md [hunk 1]
```diff
@@ -47,22 +47,22 @@ When editing `application.properties` files, you have access to:
 
 The following settings are supported:
   
+* `quarkus.tools.alwaysShowWelcomePage` : Determines whether to show the welcome page on extension startup.
 * `quarkus.tools.debug.terminateProcessOnExit` : Determines whether to terminate the quarkus:dev task after closing the debug session.
+* `quarkus.tools.formatting.surroundEqualsWithSpaces` : Insert spaces around the equals sign when formatting the application.properties file. Default is `false`.
 * `quarkus.tools.trace.server` : Trace the communication between VS Code and the Quarkus Language Server in the Output view.
 * `quarkus.tools.symbols.showAsTree` : Show Quarkus properties as tree (Outline). Default is `true`.
+* `quarkus.tools.starter.showExtensionDescriptions`: Determines whether to show the Quarkus extension descriptions when selecting Quarkus extensions. Default is `true`.
```
Comment (fbricon) [inline README.md]: move to Since 1.3.0 section

**Review by fbricon — CHANGES_REQUESTED** [review]
Except for the small change to make in the readme, this is excellent, nice work @xorye!

**Comment by xorye** [comment]
@fbricon thank you! Fixed

**Pull request merged**